### PR TITLE
DSPDC-238 Add a "smoke test" mode to the orchestration pipeline

### PR DESCRIPTION
**Pull request by danxmoran**
Tested on a run that succeeded, but the plumbing is the same for the failure and error cases so I think it's ready for review.

### CODE REVIEW

File: orchestration/templates/cron-workflow.yaml [hunk 1]
```diff
@@ -24,3 +27,26 @@ spec:
...
+                  name: {{ .Values.smokeTest.slackUrl.secretName }}
+                  key: {{ .Values.smokeTest.slackUrl.secretKey }}
+          command: [curl]
+          args:
+            - -XPOST
+            - -H
+            - 'Content-type: application/json'
+            - -d
+            - '{"text": "Workflow $(WORKFLOW_ID) entered state: $(WORKFLOW_STATE)"}'
+            - $(SLACK_URL)
```
Comment (danxmoran) [inline orchestration/templates/cron-workflow.yaml:51]: Slack's newest webhooks mechanisms tie the target channel to the URL, so the URL is all we need here.

File: orchestration/templates/cron-workflow.yaml [hunk 2]
```diff
@@ -24,3 +27,26 @@ spec:
...
+                secretKeyRef:
+                  name: {{ .Values.smokeTest.slackUrl.secretName }}
+                  key: {{ .Values.smokeTest.slackUrl.secretKey }}
+          command: [curl]
+          args:
+            - -XPOST
+            - -H
+            - 'Content-type: application/json'
+            - -d
+            - '{"text": "Workflow $(WORKFLOW_ID) entered state: $(WORKFLOW_STATE)"}'
```
Comment (danxmoran) [inline orchestration/templates/cron-workflow.yaml:50]: We can do all sorts of fancy things w/ message formatting here, but I wasn't sure what would be useful. Any ideas?

**Comment by danxmoran** [comment]
See #monster-ci in Slack for the test message.

File: orchestration/templates/cron-workflow.yaml (hunk 2 above)
Comment (snwessel) [reply orchestration/templates/cron-workflow.yaml:50]: The `WORKFLOW_ID` would be something like `encode-ingest` right? If so, what you have makes sense to me.

Comment (quazi-broad) [reply orchestration/templates/cron-workflow.yaml:50]: Hmm I'm wondering what other data points at the workflow level besides name and status?

**Pull request merged**